src/bfx_tools/isovar.ml

```diff
@@ -0,0 +1,66 @@
+open Biokepi_run_environment
+open Common
+
+module Configuration = struct
+
+  type t = {
+    name: string;
+    min_reads: int;
+    protein_sequence_length: int;
+    parameters: (string * string) list;
+  }
+  let to_json {name; min_reads; protein_sequence_length; parameters}: Yojson.Basic.json =
+    `Assoc [
+      "name", `String name;
+      "min_reads", `Int min_reads;
+      "protein_sequence_length", `Int protein_sequence_length;
+      "parameters",
+      `Assoc (List.map parameters ~f:(fun (a, b) -> a, `String b));
+    ]
+  let render {parameters; _} =
+    List.concat_map parameters ~f:(fun (a,b) -> [a; b])
+
+  let default =
+    {name = "default"; min_reads = 2; protein_sequence_length = 30; parameters = []}
+
+  let name t = t.name
+end
+
+let run ~(run_with: Machine.t)
+  ~configuration
+  ~reference_build ~vcf ~bam ~output_file =
+  let open KEDSL in
+  let isovar_tool =
+    Machine.get_tool run_with Machine.Tool.Definition.(create "isovar")
+  in
+  let genome = Machine.(get_reference_genome run_with reference_build)
+    |> Reference_genome.name
+  in
+  let min_reads = configuration.Configuration.min_reads in
+  let protein_sequence_length = configuration.Configuration.protein_sequence_length in
+  let name = sprintf "isovar_%s" (Filename.basename output_file) in
+  let command_parts = [
+    "isovar-protein-sequences.py";
+    "--vcf"; vcf#product#path;
+    "--bam"; bam#product#path;
+    "--genome"; genome;
+    "--min-reads"; string_of_int min_reads;
+    "--protein-sequence-length"; string_of_int protein_sequence_length;
```

Comment:
Not a big deal but these 2 options could make more sense in `Configuration.render`.